lib/utils/angularjs.js

```diff
@@ -53,10 +53,9 @@ export function updateElement(element, props = {}, hooks) {
   const updater = function ($rootScope) {
     const $scope = $element.scope();
 
-    Object.assign($scope, props);
-
-    $scope.$digest();
-    $rootScope.$digest();
```

Comment:
Given that the `$rootScope` is no longer needed, it can be removed from the function parameters and from `$inject`.

Reply:
Thanks. I've removed it now.